src/main/isLessThan.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+@ValidatorConstraint()
+export class IsLessThanConstraint implements ValidatorConstraintInterface {
+  validate (value: any, args?: ValidationArguments) {
+    if (value === undefined || args === undefined) {
```

Comment:
I do not think we should say valid when args are not provided. I think these are always provided by the validation framework. Having `args === undefined` is misleading at least for me.